phpunit.xml

```diff
@@ -6,7 +6,7 @@
     </testsuites>
     <php>
         <var name="NEO_USER" value="neo4j"/>
-        <var name="NEO_PASS" value="nothing"/>
+        <var name="NEO_PASS" value="test"/>
```

Comment:
you should set override NEO_PASS with phpunit run configuration and not here

Reply:
Thanks, I forgot to remove it!

That's a great tip